biomaj/workflow.py

```diff
@@ -1685,14 +1689,19 @@ def wf_uncompress(self):
                     tmpFileNameElts[len(tmpFileNameElts) - 1] = 'tmp_' + tmpFileNameElts[len(tmpFileNameElts) - 1]
                     tmpCompressedFile = self.session.get_offline_directory() + '/' + '/'.join(tmpFileNameElts)
                     archives.append({'from': origFile, 'to': tmpCompressedFile})
+                    logging.debug('Workflow:wf_tmpCompressedFile' + tmpCompressedFile)
```

Comment:
will be deleted anyway, not sure this is useful

Reply:
Yep, you're right. It was useful for my debug.